### DISCUSSION TITLE
Condition stun ist still visible after monsters next turn

### DISCUSSION THREAD

**Discussion by keinkurt** [Lurkars/gloomhavensecretariat#363] [Q&A] [answered]
I noticed, that the condition stun on a monster is still visible, but more transparent, if the monster has completed their next full turn. The condition will disapear at the end of the round.

Is that the intended behavoir? And when why?

Thank,
Christian

**Comment by Lurkars** [comment, marked as answer]
Hey, yes this is on purpose. There are some cases (or even more I didn't think of 🤷) where it's better to remember, that a Monster (standee) was stunned in that round. Obvious example: Shield ability! A stunned monster does not perform any ability. So shield does not apply to that one. How do you remember which of the standees doesn't have the shield of ability applied when stun removed? 

(Notice that this was developed before the shield/retaliate indicators on standees, but since this is an extra feature anyways and there might be other ability where this is important, I keep that feature!)
